docs/language/ql-handbook/about-the-ql-language.rst

```diff
@@ -56,14 +59,12 @@ Here are a few prominent conceptual and functional differences between general p
 -  QL's set-based semantics makes it very natural to process collections of values without having to worry about efficiently storing, indexing and traversing them.
 -  In object oriented programming languages, instantiating a class involves creating an object by allocating physical memory to hold the state of that instance of the class. In QL, classes are just logical properties describing sets of already existing values.
 
-These topics are discussed in detail in the `QL language handbook <https://help.semmle.com/QL/ql-handbook/index.html>`__.
-
-References
-----------
+Further reading
+---------------
 
 Academic references available from the `Semmle website <https://help.semmle.com/publications.html>`__ also provide an overview of QL and its semantics. Other useful references on database query languages and Datalog:
```

Comment:
Probably should have updated this link name while we were updating the link. It doesn't really make sense now that it's all on one site.
```suggestion
`Academic references <https://help.semmle.com/publications.html>`__ also provide an overview of QL and its semantics. Other useful references on database query languages and Datalog:
```